From: Ben Hutchings <ben@decadent.org.uk>
Subject: [PATCH] ALSA: Restore snd_pcm_lib_mmap_noncached()

This function is useless and was removed in 2.6.38.3, but we should
keep the export so we don't have to bump the ABI version right away.

--- a/sound/core/pcm_native.c
+++ b/sound/core/pcm_native.c
@@ -3201,6 +3201,14 @@
 EXPORT_SYMBOL(snd_pcm_lib_mmap_iomem);
 #endif /* SNDRV_PCM_INFO_MMAP */
 
+/* mmap callback with pgprot_noncached */
+int snd_pcm_lib_mmap_noncached(struct snd_pcm_substream *substream,
+			       struct vm_area_struct *area)
+{
+	return snd_pcm_default_mmap(substream, area);
+}
+EXPORT_SYMBOL(snd_pcm_lib_mmap_noncached);
+
 /*
  * mmap DMA buffer
  */
